grafana-plugin/.eslintrc.js

```diff
@@ -37,7 +37,9 @@ module.exports = {
       },
     ],
     'no-console': ['warn', { allow: ['warn', 'error'] }],
-    'no-unused-vars': [
+    'no-unused-vars': 'off',
+    'unused-imports/no-unused-imports': ['warn'],
+    'unused-imports/no-unused-vars': [
```

Comment:
is this one a superset of  `no-unused-vars`?

Reply:
Nope, it's an alternative and to make it work properly you need to disable the original `no-unused-vars`. Docs here: https://www.npmjs.com/package/eslint-plugin-unused-imports#usage